src/test/core/aztec/encoder/EncoderTest.spec.ts

```diff
@@ -776,8 +779,6 @@ describe('EncoderTest', () => {
   }
 
   function testHighLevelEncodeString(s: string, expectedBits: string | number) {
-    const jhu = StringUtils.getBytes(s, StandardCharsets.ISO_8859_1);
```

Comment:
Could you please comment the line instead of removing it? It may serve as reference as future debugging, like: 
```typescript
// TYPESCRIPTPORT: removed follwoing line because of...
// const jhu = StringUtils.getBytes(s, StandardCharsets.ISO_8859_1);
```